src/composables/useSeek/index.ts

```diff
@@ -44,6 +46,30 @@ export function useSeek(): UseSeekReturn {
     return foundChild
   }
 
+  /**
+ * Returns an array of child objects of the parent given a property
+ *
+ * @param {(THREE.Scene | THREE.Object3D)} parent
+ * @param {string} property
+ * @param {string} value
+ * @return {*}  {(THREE.Object3D[])}
+ */
+  function seekAll(parent: THREE.Scene | THREE.Object3D, property: string, value: string): THREE.Object3D[] {
+    const foundChildren: THREE.Object3D[] = []
+
+    parent.traverse((child) => {
+      if ((child as any)[property] === value) {
```

Comment:
Since we are looking by search term 'Cloth' for example, if we have objects with names 'Cloth001', 'Cloth002' this evaluation would not grab them.

You can use `includes` for this
```
if ((child as any)[property].includes(value)) {`
```